packages/site/src/routes/+page.svelte

```diff
@@ -24,7 +24,7 @@
   <CallToAction />
   <div class="sideby">
     {#if nextEvent}
```

Comment:
```suggestion
    {#if upcomingEvents.length > 0}
```

You can delete `nextEvent`!